packages/fxa-payments-server/src/components/PaymentLegalBlurb/index.scss

```diff
@@ -1,14 +1,15 @@
+@import '../../../../fxa-content-server/app/styles/variables';
+
 .payment-legal-blurb {
   clear: both;
-  color: #686869;
-  font-size: 12px;
+  color: $grey-50;
```

Comment:
Working on #9366 , I realized that the CSS variables we have don't map to the CSS variables being used in Figma. `$grey-50` is actually much lighter compared to the hex value provided in the MVP designs.

Can you just put the literal hex value here for now? #3D3D3D.

I filed https://mozilla-hub.atlassian.net/browse/FXA-3850 to fix the discrepancy.